src/views/Home/components/Schedule/index.view.tsx

```diff
@@ -1,8 +1,135 @@
-import React from "react";
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { useState } from "react";
+import * as React from "react";
 import "./schedule.scss";
+import cx from "classnames";
 
+const scheduleItems = [
+  {
+    date: "Friday, January 15, 2021",
+    events: [
+      { event: "Event Begins", time: "5:00 pm" },
```

Comment:
Can we change `event` to `name` to avoid having to call something like `event.event`?